cmake/FindLIBXSMM.cmake

```diff
@@ -1,59 +1,58 @@
-include(FindPackageHandleStandardArgs)
-
-find_path(LIBXSMM_INCLUDE_DIRS libxsmm.h
-        PATH_SUFFIXES include
-        DOC "Directory where libxsmm header files are located")
-find_library(LIBXSMM_LIBRARIES
-        NAMES xsmm
-        PATH_SUFFICES lib
-        DOC "Path to libxsmm library"
-        )
-
-find_library(LIBDL_LIBRARIES
-        NAMES dl
-        PATH_SUFFICES lib
-        DOC "Path to libdl library"
-        )
-
-list(APPEND LIBXSMM_LIBRARIES ${LIBDL_LIBRARIES})
-
-if (NOT(
-        LIBXSMM_INCLUDE_DIRS STREQUAL "LIBXSMM_INCLUDE_DIRS-NOTFOUND" OR
-        LIBXSMM_LIBARIES STREQUAL "LIBXSMM_LIBRARIES-NOTFOUND"))
-    add_library(LIBXSMM::LIBXSMM INTERFACE IMPORTED)
-    set_target_properties(LIBXSMM::LIBXSMM PROPERTIES
-            INTERFACE_LINK_LIBRARIES "${LIBXSMM_LIBRARIES}"
-            INTERFACE_INCLUDE_DIRECTORIES "${LIBXSMM_INCLUDE_DIRS}")
+if(LIBXSMM_INCLUDE_DIRS AND LIBXSMM_INCLUDE_DIRS AND LIBXSMM_LIBRARIES AND LIBXSMM_VERSION)
```

Comment:
What happens if all of them are `-NOTFOUND`?

Reply:
That case should be handled by find_package_handle_standard_args